tests/run_integration_tests.sh

```diff
@@ -4,8 +4,7 @@ set -exuo pipefail
 
 # Configuration
 WORK_DIR="/tmp/llama-stack-integration-tests"
-INFERENCE_MODEL="${INFERENCE_MODEL:-Qwen/Qwen3-0.6B}"
-EMBEDDING_MODEL="${EMBEDDING_MODEL:-ibm-granite/granite-embedding-125m-english}"
+EMBEDDING_MODEL="${EMBEDDING_MODEL:-sentence-transformers/ibm-granite/granite-embedding-125m-english}"
```

Comment:
This `sentence-transformers/ibm-granite/granite-embedding-125m-english` default is hard-coded twice: here, and above in `.github/workflows/redhat-distro-container.yml`.

My preference would be to delete it from `.github/workflows/redhat-distro-container.yml`, because I bias towards storing more logic in portable scripts than in GH's YAML.